Dockerfile

```diff
@@ -1,5 +1,5 @@
-FROM jhuopensource/semesterly-base:latest
-
+FROM sgerli/horariotec-base:latest
```

Comment:
we should not be using their base as changes there can break our prod build in the future. To make the build pass, I think you should use your own account like you were, once you have a working base image, I'd be happy to push that to jhuopensource but as a different image so we don't break the 2.x builds, paybe semesterly-py3-base